src/components/Main.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Droppable } from 'react-beautiful-dnd';
+import MainCss from '../styles/Main.module.scss';
+import MenuLine from './MenuLine';
+
+export default function Main(
+    {
+        tempText, changeText, addItem, taskLines,
+    },
+) {
+    return (
+        <div className={MainCss.main}>
+            <div className={MainCss.logo}>TODO</div>
+            <input
```

Comment:
Move it to separate `Input` component, please

Reply:
done